huaweicloud/services/lts/resource_huaweicloud_lts_cross_account_access.go

```diff
@@ -0,0 +1,300 @@
+package lts
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+// @API LTS DELETE /v3/{project_id}/lts/access-config
+// @API LTS POST /v2.0/{project_id}/lts/createAgencyAccess
+// @API LTS POST /v1/{project_id}/{resource_type}/{resource_id}/tags/action
+// @API LTS POST /v3/{project_id}/lts/access-config-list
+func ResourceCrossAccountAccess() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceCrossAccountAccessCreate,
+		UpdateContext: resourceCrossAccountAccessUpdate,
+		ReadContext:   resourceCrossAccountAccessRead,
+		DeleteContext: resourceHostAccessConfigDelete,
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the name of the cross account access.`,
+			},
+			"log_agencystream_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the log stream ID that already exists in the delegated account.`,
+			},
+			"log_agencystream_name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the log stream name that already exists in the delegated account.`,
+			},
+			"log_agencygroup_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specify the log group ID that already exists in the delegated account.`,
+			},
+			"log_agencygroup_name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specify the log group name that already exists in the	delegated account.`,
```

Comment:
so many extra spaces, check the other places

Reply:
done